engine/storage/datamotion/src/org/apache/cloudstack/storage/motion/StorageSystemDataMotionStrategy.java

```diff
@@ -207,13 +205,9 @@ public void copyAsync(DataObject srcData, DataObject destData, Host destHost, As
                     }
                 }
 
-                if (canHandleSrc) {
-                    String errMsg = "This operation is not supported (DataStoreCapabilities.STORAGE_SYSTEM_SNAPSHOT " +
-                            "not supported by destination storage plug-in). " + getDestDataStoreMsg(destData);
-
-                    LOGGER.warn(errMsg);
-
-                    throw new UnsupportedOperationException(errMsg);
+                if (canHandleDest) {
```

Comment:
Same here. I was trying to rebase to the master which got ugly really quickly then I reverted back to the old code and somewhere in between I lot the commit. I will fix this